hibernate-core/src/main/java/org/hibernate/dialect/AbstractTransactSQLDialect.java

```diff
@@ -40,11 +40,6 @@
 	public AbstractTransactSQLDialect() {
 		super();
 
-		registerColumnType( Types.BOOLEAN, "bit" );
```

Comment:
Removing this line breaks `SQLServerDialect` and `SybaseAnywhereDialect`, I believe.

Really I wish you wouldn't mess too much with these type mappings because I put a *lot* of work into fixing them, and the truth is I just don't remember all the details of every mapping off the top of my head.

If you want to change type mappings, please do it in *small* pull requests, please don't roll changes like that into a big "omnibus" PR.

Reply:
Oh, I see you also made changes to the subclasses. But it's still a change to behavior on Sybase Anywhere that I don't think we want. Whatever, such a change doesn't belong in *this* pull request.

Reply:
Ok I removed these changes form this PR and will create a separate one or discuss on your PR